grails-app/services/org/pih/warehouse/importer/LocationImportDataService.groovy

```diff
@@ -107,7 +107,7 @@ class LocationImportDataService implements ImportDataService {
             }
 
             // Do not allow to change internal type location to non-internal
-            if (params.locationType && location && (location.locationType.isInternalLocation() || location.locationType.isZone()) && !(LocationType.findByNameLike(params.locationType + "%").isInternalLocation() || LocationType.findByNameLike(params.locationType + "%").isZone())) {
+            if (params.locationType && location && (location?.locationType?.isInternalLocation() || location?.locationType?.isZone()) && !(LocationType.findByNameLike(params.locationType + "%")?.isInternalLocation() || LocationType.findByNameLike(params.locationType + "%")?.isZone())) {
```

Comment:
the `&& location` part makes adding `location?` unnecessary, right?

Reply:
yup